web-bff/app/utils/logging_config.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import logging.config
+from pathlib import Path
+
+# Create logs directory if not exists
+LOG_DIR = Path("logs")
+LOG_DIR.mkdir(exist_ok=True)
+
+# Logging Configuration
+LOGGING_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "default": {"format": "%(asctime)s - %(levelname)s - %(name)s - %(message)s"},
+        "detailed": {
+            "format": "%(asctime)s - %(levelname)s - %(name)s - %(filename)s - %(lineno)d - %(message)s"
+        },
+    },
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "formatter": "default",
+            "level": "DEBUG",
+        },
+        "file": {
+            "class": "logging.FileHandler",
+            "filename": LOG_DIR / "app.log",
```

Comment:
Some/many of these need to be configurable from an external file and/or env var. Having them hardcoded into the compiled code is an ANTI-PATTERN (read about 12-factor apps)